Makefile

```diff
@@ -0,0 +1,32 @@
+VERSION?=0.0.0
+
+.PHONY: build
+build: build-py
+	@echo "Built $(VERSION)"
+
+.PHONY: build-py
+build-py:
+	@echo "Building Python package" && \
+	cp data/public.gpg python/insights_ansible_playbook_verifier/data/public.gpg
+	cp data/revoked_playbooks.yml python/insights_ansible_playbook_verifier/data/revoked_playbooks.yml
+	sed -i "s|version = .*|version = '$(VERSION)'|" pyproject.toml
+
+
+.PHONY: test
+test: test-py
+
+.PHONY: test-py
+test-py:
+	PYTHONPATH=python/ pytest python/tests-unit/ -v
+	PYTHONPATH=python/ pytest python/tests-integration/ -v
```

Comment:
hmm maybe this needs to be removed, as the functional tests now use an already-installed tools; maybe move it to a separate makefile target? we will run them via packit/testing-farm later on